image/templates/helm/stackrox-central/templates/99-openshift-monitoring.yaml

```diff
@@ -92,4 +92,25 @@ subjects:
     name: central
     namespace: "{{ .Release.Namespace }}"
 
+---
+
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: "central-telemeter-{{ .Release.Namespace }}"
+  namespace: openshift-monitoring
```

Comment:
like for ServiceMonitor objects, it would be better to deploy the rule in your controlled namespace.

Reply:
Unfortunately that's not feasible for ACS because with our current installation methods we don't own the namespace where Central is installed into. So we cannot reliably set the namespace label required for OCP platform monitoring. So while not ideal I think `openshift-monitoring` is the best place for it.